READMEs/wallets.md

```diff
@@ -31,7 +31,7 @@ Here's how. In Python:
 
 ```python
 import web3
-private_key = web3.Web3().eth.account.create().privateKey
+private_key = web3.Web3().eth.account.create().key
```

Comment:
This is not a change from camelCase -> snake case, it is a change in the usage of a 3rd party tool's API. Don't do it here. If you _do_ want to do it somewhere, put it in a different issue + PR and explain why you're making the change.

Reply:
@mariacarmina can you please attach the relevant documentation? Has the new version of web3 equated these properties? (I doubt it, maybe it's private_key... But if the web3 version did deprecate privateKey and replaced it with "key", I think we can handle that separately, in #342)

Reply:
@trentmc while I agree there is some overlap between this ticket and the deprecation one, I would rather keep them separate. However, in some cases because the new web3 version deprecated camelCase and replaced it with snake_case, we are fixing two things at once. I have no issue with this overlap, if not blatant. Switching between branches is exhausting, especially for small changes such as casing.

Reply:
Agreed. And: If it's two things at once, then the PR name and github issue should be for the more important issue. Pointing to an updated API is the more important thing. We shouldn't do it as a side effect of something cosmetic like snake case.

Reply:
Addressed here: https://github.com/oceanprotocol/ocean.py/pull/342/files#r647164717
It can be removed from this PR.

Reply:
I have also did the same modification in the #342 and I will attach here a relevant snapshot from the code, because that was my documentation at that moment.
![Screenshot from 2021-06-08 10-08-54](https://user-images.githubusercontent.com/50501033/121139620-9ef1ac80-c841-11eb-95c7-36b65ff47eae.png)

Reply:
The path for that is `venv/lib/python3.8/site-packages/eth_account/signers/local.py`

Reply:
Here is the demo :)
![Screenshot from 2021-06-08 10-39-43](https://user-images.githubusercontent.com/50501033/121143900-2d682d00-c846-11eb-80fe-311a03d568f3.png)

Reply:
If the change is already being done in another issue / PR then why is it still in this PR?

Reply:
I discovered the deprecated function in this PR and after I modified in #342 , that was the reason, now I think it's all good. I have modified in the right place.